actors/common/src/main/java/org/sunbird/learner/util/DataCacheHandler.java

```diff
@@ -119,7 +120,7 @@ private void cache(Map<String, Map<String, Object>> map, String tableName) {
       ProjectLogger.log("pagemap keyset " + map.keySet());
       ProjectLogger.log(tableName + " cache size: " + map.size(), LoggerEnum.INFO.name());
     } catch (Exception e) {
-      ProjectLogger.log(e.getMessage(), e);
+      ProjectLogger.log("DataCacheHandler:cache page section " + e.getMessage(), e);
```

Comment:
From:
"DataCacheHandler:cache page section "

To:
"DataCacheHandler:cache: Exception in retrieving page section "